### ISSUE TITLE
Replace libcompose with compose-go

### ISSUE THREAD

**Issue by keviiin38** [kubernetes/kompose#1466]
As [libcompose](https://github.com/docker/libcompose) is now **deprecated** in favor of [compose-go](https://github.com/compose-spec/compose-go), is a migration planned or possible ?

_I'd love to contribute this feature, but don't know how complex this migration can be ? How long can it take ?_
_Saw the open PR https://github.com/kubernetes/kompose/pull/1440  about this_

**Comment by hangyan** [comment]
Yes, there are still plans to support this. Im not sure if you are still intreseted for now.  The current PR is opened by compose-go guys and they dont have much time to finish it. So I guess contibution is welcome.

**Comment by hangyan** [comment]
If you are still intreseted, feel free to open a new PR. Thanks for the contibution.
